src/GraphQL.Tests/Bugs/Bug947.cs

```diff
@@ -63,9 +63,7 @@ public void GetArgument_Should_Return_Properly_Converted_Values()
         Should.Throw<InvalidOperationException>(() => context.GetArgument<string>("object"));
         Should.Throw<InvalidOperationException>(() => context.GetArgument<DateTime>("object"));
 
-        var otherObject = context.GetArgument<SomeOtherObject>("object");
-        otherObject.unknown.ShouldBe(0);
-        otherObject.unknown2.ShouldBeNull();
+        Should.Throw<InvalidOperationException>(() => context.GetArgument<SomeOtherObject>("object"));
```

Comment:
This should never have worked.  `inner_int` doesn't exist on `SomeOtherObject` and now properly throws an exception.